src/styles/index.css

```diff
@@ -0,0 +1,8 @@
+@import "./reset.css";
+@import "./blocks/page.css";
+@import "./blocks/img.css";
+@import "./blocks/title.css";
+@import "./blocks/code.css";
+@import "./blocks/stars.css";
+@import "./blocks/price.css";
+@import "./blocks/link.css";
```

Comment:
I'm not sure we need to split card styles to multiple files like this

Reply:
11. [FILE STRUCTURE] - You should create separate file for each BEM block

Reply:
Yes sure, but you created separate files for each BEM element